tests/templates/test_subroutines/test_qubitization.py

```diff
@@ -255,10 +257,9 @@ def test_qnode_jax(self, shots, use_jit, device):
         assert jac.shape == (4,)
         assert np.allclose(jac, self.exp_grad, atol=0.05)
 
+    # TODO: finite shots fails because Prod is not currently differentiable.
     @pytest.mark.torch
-    @pytest.mark.parametrize(
-        "shots", [None]
-    )  # TODO: finite shots fails because Prod is not currently differentiable.
+    @pytest.mark.parametrize("shots", [None])
```

Comment:
Maybe add finite shots to the parametrization but `xfail` inside the test if `shots is not None`